### DISCUSSION TITLE
keep `<plug>` mapping not remove

### DISCUSSION THREAD

**Discussion by milanglacier** [Vigemus/iron.nvim#261] [General]
Personally I will map multiple keymaps to one execution,
for example I will map `<Leader>ss`, `<Leader>sl` to same execution: `iron send line`,
so deprecating `<plug>` mapping will make my config be less clearer,
I next need to call `mappings = {send_a_line = '<Leader>ss'}` first, and then calling `nmap <Leader>ss <Leader>sl`,
i.e a recursive map, which is not ideal.
For a `<plug>` style mapping, I can map `<leader>ss` and `<Leader>sl` to `<Plug>iron-send-a-line` which is definetely more clear

**Comment by stephenprater** [comment]
I agree with this - I hate figuring out a different way to do keymaps for every plugin.  Please continue to provide `<plug>` mappings so we can use Vimscript mappings.
